### PR TITLE
Add Redmonk blog post

### PR DESCRIPTION
**Pull request by cnunciato**
Fixes https://github.com/pulumi/marketing/issues/289.

### CODE REVIEW

File: themes/default/content/blog/redmonk-pulumi-developer-first-infrastructure/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+1. [Cloud Engineering]({{< relref "/cloud-engineering" >}})
+1. Harmonized Applications and Infrastructure
+1. Beyond "Lift and Shift"
+1. Secure and Observable By-Construction
+1. An API for Everything
+
+![Principles of developer-first infrastructure](./developer-first-infra-principles.png)
+
+Finally, Joe dives into the best practices of developer-first infrastructure:
```
Comment (gthuang) [inline themes/default/content/blog/redmonk-pulumi-developer-first-infrastructure/index.md:43]: can you add a little more spacing between this row and the screenshot that appears before it? it feels a little too tight and it's easy to skim past this.

File: themes/default/content/blog/redmonk-pulumi-developer-first-infrastructure/index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+1. **Continuous Verification**: Verify and enforce guardrails at all times.
+1. **Environment Parity**: Share as much as possible between dev, test, and prod environments.
+1. **Securely Configured**: Loosely couple and securely configure environments.
+1. **Distributed Architectures**: Go beyond "lift and shift" to building truly cloud-native applications and infrastructure.
+1. **Observable**: Instrument applications and infrastructure to log high cardinality events, early and often.
+
+## Next steps
+
+Developer-first infrastructure means empowering developers to build and deploy modern cloud applications and infrastructure through the use of software engineering practices that tame modern cloud complexity. If you feel inspired after viewing the Pulumi/RedMonk discussion video, then consider applying developer-first principles and practices for your next greenfield project or application using an infrastructure as code tool like Pulumi. You can also [watch Joe Duffy's presentation](https://www.youtube.com/watch?v=SQRM0r5U1js) on developer-first infrastructure from the Cloud Engineering Summit.
```
Comment (gthuang) [inline themes/default/content/blog/redmonk-pulumi-developer-first-infrastructure/index.md:61]:
```suggestion
Developer-first infrastructure means empowering developers to build and deploy modern cloud applications and infrastructure through the use of software engineering practices that tame modern cloud complexity. If you feel inspired after viewing the Pulumi/RedMonk discussion video, then consider applying developer-first principles and practices for your next greenfield project or application using an infrastructure as code tool like Pulumi. You can also [watch Joe Duffy's presentation](https://www.youtube.com/watch?v=SQRM0r5U1js) on developer-first infrastructure from the Cloud Engineering Summit to learn more about developer-first.
```

Comment (gthuang) [reply themes/default/content/blog/redmonk-pulumi-developer-first-infrastructure/index.md:61]: can you also link the reference to "Pulumi" in the 2nd to last sentence to our getting started docs?

**Review by gthuang — CHANGES_REQUESTED** [review]

File: themes/default/content/blog/redmonk-pulumi-developer-first-infrastructure/index.md (hunk 1 above)
Comment (cnunciato) [reply themes/default/content/blog/redmonk-pulumi-developer-first-infrastructure/index.md:43]: Not in our current styling, alas; we apply these same margins everywhere. I could add some padding to the images themselves, though.

**Review by gthuang — APPROVED** [review]

**Pull request merged**